lib/health_check/middleware_health_check.rb

```diff
@@ -7,7 +7,8 @@ def initialize(app)
 
     def call(env)
       uri = env['PATH_INFO']
-      if uri.include? HealthCheck.uri
+      uriWithoutSuffix = uri[/([^\.]*)/,1]
```

Comment:
or event
URI_SUFFIX_REGEX = /\.[\w]{2,}$/
then in call method
uri_without_suffix = uri.gsub URI_SUFFIX_REGEX, ''